submissions/cynthiaNwakaeme/domManipulation/shippingCost.js

```diff
@@ -0,0 +1,53 @@
+let quantity = document.getElementById("quantityNumber"),
+    price = document.getElementById("price"),
+    shippingCost = document.getElementById("shippingCost"),
+    totalCost = document.getElementById("totalCost"),
+    button = document.getElementById("buyNow");
+
+
+
+let shippingAnalysis = (event) => {
+    let integerRegex = /[^0-9]/g; //To ensure only integers are allowed
+
+    if (quantity.value == "" || quantity.value < 1 || quantity.value.match(integerRegex)) {
+        button.disabled = true;
+        button.style.backgroundColor = "rgba(218, 165, 32, 0.55)";
+        shippingCost.innerHTML = 0;
+        totalCost.innerHTML = 0;
+        event.preventDefault();
+        return false;
+
+    } else {
```

Comment:
I don't think you should prevent default button submit on this else block. And you should also `return true` here

Reply:
Thank You ma.

Reply:
I think I confused you. Please return true in the else block and false in the if block. I am sorry about that